cli/spec/kontena/cli/stacks/labels/add_command_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'kontena/cli/stacks/label_command'
+require 'kontena/cli/stacks/labels/add_command'
+
+describe Kontena::Cli::Stacks::Labels::AddCommand do
+  include ClientHelpers
+  include OutputHelpers
+
+  # stack name
+  let :name do
```

Comment:
Other specs tend to have parenthesis and the oneliner syntax for these: 

```ruby
let(:name) { 'test-stack' }
```

Reply:
✅ RESOLVED